packages/billing/src/index.ts

```diff
@@ -0,0 +1,11 @@
+export {
+    split,
+    areAllRecipientsUnique,
+    createRecipientKey,
+    getVorItems,
+    hasFeePayers,
+    hasOverpaymentReceivers,
+    hasSingleVorItem,
+    sortByVorAndOrderComparator,
```

Comment:
Why do you need to export this much? Isn't the split enough?

Reply:
These functions are used outside.